src/providers/completer/citation.ts

```diff
@@ -43,6 +43,7 @@ export class Citation {
             item.filterText = `${item.key} ${item.fields.author} ${item.fields.title} ${item.fields.journal}`
             item.insertText = item.key
             item.documentation = new vscode.MarkdownString(item.detail.replace(/\n/g, '  \n'))
+            item.detail = ''
```

Comment:
@tamuratak What do you think of dropping `item.detail`?
It was rendered with a monospace font, whereas `item.documentation` is now markdown and is rendered using a much smaller font.

Reply:
I do not have a specific opinion. 

`item.detail` will be always visible. See microsoft/vscode/issues/39441. It might be used for another purpose.

Reply:
Thanks for pointing microsoft/vscode/issues/39441. For the moment, `item.detail` and `item.documentation` are always duplicates of each other, so I will keep `item.detail = ''` and I will come back to this once the `API` for `completionItem` is changed.